loki_role/tasks/grafana.yaml

```diff
@@ -0,0 +1,63 @@
+---
+- name: Install prerequisite packages on Debian-based systems
+  ansible.builtin.apt:
+    name:
+      - gnupg
+      - curl
+    state: present
+  when: ansible_facts['os_family'] == 'Debian'
```

Comment:
As we discussed earlier as well variables the package name don't hardcode it inside the role